projects/zkdx/index.js

```diff
@@ -0,0 +1,22 @@
+const sdk = require("@defillama/sdk");
+const ADDRESSES = require('../helper/coreAssets.json')
+const {staking} = require("../helper/staking");
+const {gmxExports} = require("../helper/gmx");
+
+module.exports = {
+    timetravel: false,
+    misrepresentedTokens: false,
+    methodology: 'zkDX counts the vault and staking values as tvl',
+    start: 3744214,
+    era: {
+        tvl: sdk.util.sumChainTvls(
+            [
+                gmxExports({vault: "0x6a29d14176248E84760473d28973F53821fB6287"}),
+                staking(
```

Comment:
why is staking included in derivatives tvl? and why are derivatives numbers this? 
![image](https://github.com/DefiLlama/DefiLlama-Adapters/assets/99907941/02a66741-f6d5-4def-bc53-75b5c9840c8b)